n2y/notion.py

```diff
@@ -61,15 +61,17 @@ def wrapper(*args, **kwargs):
             client.retry_count = 0
             return response
         except APIResponseError as err:
-            can_retry = 'retry-after' in err.headers
-            if can_retry and client.retry_api_calls:
+            rate_limited = err.code == 'rate_limited'
+            retry_after = 'retry-after' in err.headers
```

Comment:
This variable naming is a bit confusing to me. `retry_after` is a boolean but we also later use it as the amount to sleep by. Maybe we can just use two variables here: one as the boolean to see if we have `retry-after` in the headers and one that stores the amount to sleep by?